build.gradle

```diff
@@ -0,0 +1,28 @@
+allprojects {
+    apply plugin: 'maven'
+
+    group = 'nl.tudelft'
+    version = '1.0'
+}
+
+subprojects {
+    apply plugin: 'java'
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        jcenter()
+    }
+
+
+    dependencies {
+        compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.6.2'
+        testCompile group: 'junit', name: 'junit', version: '4.12'
+        testCompile group: 'org.mockito', name: 'mockito-all', version: '2.0.2-beta'
```

Comment:
Please use `mockito-core` with a recent stable version instead. For example https://mvnrepository.com/artifact/org.mockito/mockito-core/2.15.0 is the latest published.